Slimefun Beautified/pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
    "pack":{
-      "pack_format":8,
-      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- (v1.0)"
+      "pack_format":9,
+      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e(24/10/2022 §7- &ev1.1)"
```

Comment:
Planning to replace date system with Git commit IDs. Version bump is only done before pushing out a new version.
```suggestion
      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- v1.0)"
```